Objects/object.c

```diff
@@ -2372,7 +2372,7 @@ new_reference(PyObject *op)
 #else
     op->ob_tid = _Py_ThreadId();
     op->_padding = 0;
-    op->ob_mutex = (struct _PyMutex){ 0 };
+    op->ob_mutex = (PyMutex){ 0 };
```

Comment:
If there is no public PyMutex_STATIC_INIT, can you maybe add a private one in pycore_lock.h?

Reply:
Let's consider that in a separate PR:

* It's not directly related to making the API public
* It would make sense to consider that for multiple types (e.g., `PyEvent`, `_PyOnceFlag`), not just `PyMutex`